### PR TITLE
Add new recipes, fix some data stuff

### PR DESCRIPTION
**Pull request by redcreeper14385**
Adds recipes to make the mod playable. Also fixes some dodgy JSONs.
Works on #44, although this isn't finished yet.

### CODE REVIEW

File: motherlode-copper-dungeon/src/main/resources/data/motherlode-copper-dungeon/recipes/cutter_trap.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern": [
+    " # ",
+    "cGc",
+    "MIM"
```
Comment (mschae23) [inline motherlode-copper-dungeon/src/main/resources/data/motherlode-copper-dungeon/recipes/cutter_trap.json:6]: `I`?

File: motherlode-copper-dungeon/src/main/resources/data/motherlode-copper-dungeon/recipes/zapper_trap.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern": [
+    "l l",
+    "cGc",
+    "MIM"
```
Comment (mschae23) [inline motherlode-copper-dungeon/src/main/resources/data/motherlode-copper-dungeon/recipes/zapper_trap.json:6]: Same here, there's no key for `I`

File: motherlode-redstone/src/main/resources/data/motherlode-redstone/recipes/block/redstone_transmitter.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "GRG",
+    "ccc"
+  ],
+  "key": {
+    "#": {
+      "item": "minecraft:redstone_torch"
+    },
+    "E": {
+      "tag": "motherlode-materials:gems"
+    },
```
Comment (mschae23) [inline motherlode-redstone/src/main/resources/data/motherlode-redstone/recipes/block/redstone_transmitter.json:14]: Should this be `G`?

**Review by mschae23 — COMMENTED** [review]
I like the recipes. I think there are a few small mistakes though.

File: motherlode-redstone/src/main/resources/data/motherlode-redstone/recipes/block/redstone_transmitter.json (hunk 3 above)
Comment (redcreeper14385) [reply motherlode-redstone/src/main/resources/data/motherlode-redstone/recipes/block/redstone_transmitter.json:14]: yes, not sure what happened there

**Review by mschae23 — APPROVED** [review]

**Review by marcus8448 — APPROVED** [review]

**Pull request merged**